nemo/collections/nlp/modules/dialogue_state_tracking/sgd_encoder.py

```diff
@@ -0,0 +1,77 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
+# Copyright 2019 The Google Research Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Dict, Optional
+
+from torch import nn as nn
+
+from nemo.collections.nlp.modules.common.classifier import Classifier
+from nemo.core.classes import typecheck
+from nemo.core.neural_types import ChannelType, LogitsType, NeuralType
+
+__all__ = ['SGDEncoder']
+
+ACT2FN = {"tanh": nn.functional.tanh, "relu": nn.functional.relu}
```

Comment:
Are these two activations different in results? Which one was used in the paper?

Reply:
tanh